theorems/T000768.md

```diff
@@ -0,0 +1,11 @@
+---
+uid: T000768
+if:
+  and:
+  - P000012: true
+  - P000217: true
+then:
+  P000050: true
+---
+
+If $x\in U$ where $U$ is open, then there exists zero-set $Z$ and cozero set $W$ such that $x\in Z\subseteq W\subseteq U$. Since $X$ is strongly zero-dimensional we can find clopen set $V$ such that $x\in Z\subseteq V\subseteq W\subseteq U$. So $X$ is zero-dimensional.
```

Comment:
```suggestion
If $x\in U$ where $U$ is open, then there exists zero-set $Z$ and cozero set $W$ such that $x\in Z\subseteq W\subseteq U$. Since $X$ is {P217}, we can find clopen set $V$ such that $x\in Z\subseteq V\subseteq W\subseteq U$. So $X$ is {P50}.
```